ts/features/walletV3/payment/hooks/usePaymentFailureSupportModal.tsx

```diff
@@ -117,30 +122,21 @@ const usePaymentFailureSupportModal = ({
     clipboardSetStringWithFeedback(data);
   };
 
-  const displayPhoneNumber = PAGOPA_SUPPORT_PHONE_NUMBER.replace(
-    /^(\d{2})(\d{4})(\d{4})$/,
-    "$1.$2.$3"
-  );
-
   const { bottomSheet, present, dismiss } = useIOBottomSheetAutoresizableModal({
     component: (
       <>
         <ListItemHeader label={I18n.t("wallet.payment.support.supportTitle")} />
-        {withPhoneSupport && (
-          <ListItemAction
-            label={I18n.t("wallet.payment.support.phone", {
-              phoneNumber: displayPhoneNumber
-            })}
-            accessibilityLabel={I18n.t("wallet.payment.support.phone", {
-              phoneNumber: displayPhoneNumber
-            })}
-            onPress={() =>
-              Linking.openURL(`tel:${PAGOPA_SUPPORT_PHONE_NUMBER}`)
-            }
-            variant="primary"
-            icon="phone"
-          />
-        )}
+        <ListItemAction
+          label={I18n.t("wallet.payment.support.phone", {
+            phoneNumber: "06.4520.2323"
```

Comment:
I think it is better to leave it as it was declaring the `PAGOPA_SUPPORT_PHONE_NUMBER`, maybe a merge conflict? What do you think?

Reply:
Fixed in https://github.com/pagopa/io-app/pull/5390/commits/336a54bf871c389fcea39cc1b0be327fb48a3137